PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## Description
+
+Describe the changes made and why they were made instead of how they were made.
+
+## Related issues and discussion
+#{Issue Number}
+ 
+## Screen-shots, if any
```

Comment:
Why not keep the default as  `<!-- ## Screen-shots, if any -->` (commented out) 
Since screenshots are not always added

Reply:
I would change this line to: `For user interface changes: Screenshots showing what changed`